tools/run_tests/run_xds_tests.py

```diff
@@ -1282,6 +1282,9 @@ def test_circuit_breaking(gcp, original_backend_service, instance_group,
         logger.info('UNARY_CALL reached stable state after increase (%d)',
                     extra_backend_service_max_requests)
         logger.info('success')
+        configure_client([
```

Comment:
I propose this change just as a nice-to have. This is the end of the `test_circuit_breaking` function in the Python test runner after the test case is successful. I propose we call `Configure()` once more to the test client to reset it to be only sending `UnaryCall` with empty metadata at this stage. The test runner will eventually kill the test client, but in the meantime, since the last `Configure()` call is with `rpc-behavior: keep-open`, in this particular instance the Ruby test client will keep opening new threads to send those RPCs. It takes a few minutes for the cleanup to happen so in those few minutes, the Ruby test client could have easily started tens of thousands of new threads. It did not cause any problem, per se, as the tests will still pass as those RPCs are irrelevant and the test client will be killed eventually. So this is just a nice-to-have.

Reply:
nit: it would be good to add some note. Something like

```
// Avoid new RPCs being outstanding (some test clients create threads
// for sending RPCs) after restoring backend services.
```

Reply:
Added.